### ISSUE TITLE
Use non-yet downloaded models to start a new conversation

### ISSUE THREAD

**Issue by Kerollmops** [zeozeozeo/ellama#18]
It is maybe out of scope for this project, but I would like to be able to start a conversation with a model that is not yet installed on the machine.

**Comment by zeozeozeo** [comment]
I agree, it would be cool to have something like the [Ollama library](https://ollama.com/library) built into the app, the Ollama API allows for pulling any model remotely
